### PR TITLE
F/ Design on unavailable project added

### PR DESCRIPTION
**Pull request by mateodaza**

### CODE REVIEW

File: src/components/views/project/NotAvailableProject.tsx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+		fetchSuggestedProjects();
+	}, []);
+
+	return (
+		<Wrapper>
+			<Image
+				src='/images/missing-project.svg'
+				width={122.69}
+				height={112}
```
Comment (MohammadPCh) [inline src/components/views/project/NotAvailableProject.tsx:51]: please add alt attribute :)

File: src/components/views/project/NotAvailableProject.tsx [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+			/>
+			<TitleText>Oops! This project is no longer available!</TitleText>
+			<SubtitleText>Take a look at similar projects</SubtitleText>
+			<Slider>
+				{listPosition !== 0 ? (
+					<CaretLeft
+						src='/images/caret_right.svg'
+						onClick={() => moveList(-1)}
+					/>
```
Comment (MohammadPCh) [inline src/components/views/project/NotAvailableProject.tsx:60]: we have it on the design system, I suggest using it (CaretRight, CaretLeft, and ...).

**Review by MohammadPCh — CHANGES_REQUESTED** [review]
Thank you @mateodaza ,
LGTM, just comment some suggestions.

File: src/components/views/project/NotAvailableProject.tsx (hunk 2 above)
Comment (mateodaza) [reply src/components/views/project/NotAvailableProject.tsx:60]: The design is different than the ones on the design system :c

**Review by MohammadPCh — APPROVED** [review]
Nice.  Thank you @mateodaza

**Pull request merged**